pkg/antctl/antctl.go

```diff
@@ -370,6 +368,8 @@ $ antctl get podmulticaststats pod -n namespace`,
 			long:    "Dump all the OVS flows or the flows installed for the specified entity.",
 			example: `  Dump all OVS flows
   $ antctl get ovsflows
+  Dump all OVS tables with name only
+  $ antctl get ovsflows --table-names
```

Comment:
I think the flag name doesn't match the actual name? Should it be `--table-names-only`?